packages/client/src/comp/AddressInputs.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react"
+
+import { BaseInput } from "./util/Input"
+import styled from "styled-components"
+import { cssQuery } from "./util/cssQuery"
+import { InputId } from "./states/BaseContainer"
+
+const FlexBox = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-wrap: wrap;
+  align-items: center;
+  margin: 1em -0.5em;
+`
+
+const FlexGrow = styled.div`
+  margin: 0 0.5em;
+`
+
+const FlexGrowFull = styled.div`
+  margin: 0 0.5em;
+  flex: 0.5;
+  ${cssQuery.small} {
+    flex: 1;
+  }
+`
+
+const FlexGrowHalf = styled.div`
+  margin: 0 0.5em;
+  flex: 0.25;
+  ${cssQuery.small} {
+    flex: 0.5;
+  }
+`
+
+interface Props {
```

Comment:
Can you type these correctly.  We don't use `any` for types.

In particular, fields needs to be typed as in `Base.tsx`.